person.rb

```diff
@@ -1,20 +1,22 @@
 require './main'
 require './capitalize_decorator'
 require './trimmer_decorator'
+require './rental'
 
 class Person < Nameable
```

Comment:
- [ ] Your Person class is missing the other end of the has many relationship. You need to add a method for that:

  ```
  def add_rental(book, date)
    Rental.new(date, book self)
  end
  ```